File: Lista/Java/Q1.java
class Carro {
    private String marca;
    private String modelo;
    private int ano;

    public Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    public void exibirDetalhes() {
        System.out.println("Marca: " + marca + ", Modelo: " + modelo + ", Ano: " + ano);
    }
}

public class Main {
    public static void main(String[] args) {
        Carro carro1 = new Carro("Ford", "Mustang", 2022);
        Carro carro2 = new Carro("Chevrolet", "Camaro", 2021);
        Carro carro3 = new Carro("Toyota", "Corolla", 2023);

        carro1.exibirDetalhes();
        carro2.exibirDetalhes();
        carro3.exibirDetalhes();
    }
}

File: Lista/Go/Q1.go
package main

import "fmt"

// Definindo a classe (struct) Carro
type Carro struct {
    Marca  string
    Modelo string
    Ano    int
}

// Função para exibir os detalhes do carro
func (c Carro) ExibirDetalhes() {
    fmt.Printf("Marca: %s, Modelo: %s, Ano: %d\n", c.Marca, c.Modelo, c.Ano)
}

func main() {
    // Instanciando três objetos da classe Carro
    carro1 := Carro{
        Marca:  "Toyota",
        Modelo: "Corolla",
        Ano:    2020,
    }

    carro2 := Carro{
        Marca:  "Honda",
        Modelo: "Civic",
        Ano:    2018,
    }

    carro3 := Carro{
        Marca:  "Ford",
        Modelo: "Mustang",
        Ano:    2021,
    }

    // Exibindo os detalhes de cada carro
    carro1.ExibirDetalhes()
    carro2.ExibirDetalhes()
    carro3.ExibirDetalhes()
}
